SU-HSDO/suhumsci
# SHS-5148: fix hover styles in hb-categories

# READY FOR REVIEW

## Summary
- Add padding to `hb-categories` list item to make underline on hover visible

## Need Review By (Date)
11/27

## Urgency
medium

## Steps to Test
1. Visit https://hs-colorful.suhumsci.loc/default-views/people
2. Confirm that the "Research Areas" items present in some cards have a bottom border on hover
3. Repeat test on https://hs-traditional.suhumsci.loc/default-views/people

<img width="377" alt="Screenshot 2023-11-27 at 10 28 40" src="https://github.com/SU-HSDO/suhumsci/assets/3496574/ce2f3139-a4e4-4fa7-80f6-67ec87163eec">

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
